.github/workflows/release-tagged.yml

```diff
@@ -0,0 +1,95 @@
+name: Release Tagged
+
+on:
+  repository_dispatch:
+    types: [release-tagged]
+  # TODO: Remove after done testing
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  release-tagged:
+    runs-on: ubuntu-latest
+    container:
+      image: hairyhenderson/dockerfiles-builder:latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # We disabled yq for now because of an upstream bug with whitespace
+      # See https://github.com/mikefarah/yq/issues/343
+      # - name: Install yq
+      #   run: |
+      #     sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CC86BB64
+      #     sudo add-apt-repository ppa:rmescandon/yq
+      #     sudo apt update
+      #     sudo apt install yq -y
```

Comment:
If `yq` is needed (later?) it should probably just come from the `hairyhenderson/dockerfiles-builder` image... FWIW, I've added it in https://github.com/hairyhenderson/dockerfiles/commit/ad2d2b672ee4eda14fe6cb92c40b87f5d1f666e1

Reply:
Cool - yeah I'll just remove the yq stuff for now. We can revisit it later if we feel the need to (but I think sed is perfectly fine here :man_shrugging:)